cachecontrol.go

```diff
@@ -29,15 +29,15 @@ func ParseCacheControl(input string) (CacheControl, error) {
 		var token string
 		switch input[pos] {
 		case '"':
-			token = readString(input, pos+1, "\"")
+			token = readUntil(input, pos+1, "\"")
 			pos += len(token) + 1
 		case ',', '\n', '\r', ' ', '\t':
 			continue
 		case '=':
 			isValue = true
 			continue
 		default:
-			token = readString(input, pos, "\"\n\t\r ,=")
+			token = readUntil(input, pos, "\"\n\t\r ,=")
```

Comment:
It [was in 1.7](https://github.com/golang/go/blob/go1.7.4/src/strings/strings.go#L192-L205), but it's [more involved in 1.8](https://github.com/golang/go/blob/go1.8beta2/src/strings/strings.go#L168-L191).